src/data-source.ts

```diff
@@ -0,0 +1,19 @@
+import 'reflect-metadata'
+import { DataSource } from 'typeorm'
+import { User } from './entity/User'
+import { Team } from './entity/Team'
+import { WorkPeriod } from './entity/WorkPeriod'
+
+export const AppDataSource = new DataSource({
+    type: 'postgres',
```

Comment:
Please extract the database credentials (host, port, username, password, database name) and other sensitive information to a ﻿.env file and use ﻿ConfigService to access them. This approach increases security by preventing sensitive data exposure and makes the code more flexible to changes in different environments.